### PR TITLE
fix: include package_desc.md when building packages

### PR DESCRIPTION
**Pull request by joaoandre-avaiga**

### CODE REVIEW

File: tools/packages/taipy-config/MANIFEST.in [hunk 1]
```diff
@@ -2,3 +2,4 @@ include taipy/config/*.pyi
 include taipy/config/*.json
 include *.json
 include taipy/config/setup.requirements.txt
+include package_desc.md
```
Comment (jrobinAV) [inline tools/packages/taipy-config/MANIFEST.in:5]:
```suggestion
include taipy/config/package_desc.md
```

File: tools/packages/taipy-core/MANIFEST.in [hunk 2]
```diff
@@ -2,3 +2,4 @@ include taipy/core/*.json
 include taipy/core/config/*.json
 include *.json
 include taipy/core/setup.requirements.txt
+include package_desc.md
```
Comment (jrobinAV) [inline tools/packages/taipy-core/MANIFEST.in:5]:
```suggestion
include taipy/core/package_desc.md
```

File: tools/packages/taipy-gui/MANIFEST.in [hunk 3]
```diff
@@ -4,3 +4,4 @@ include taipy/gui/viselements.json
 include taipy/gui/*.pyi
 include *.json
 include taipy/gui/setup.requirements.txt
+include package_desc.md
```
Comment (jrobinAV) [inline tools/packages/taipy-gui/MANIFEST.in:7]:
```suggestion
include taipy/gui/package_desc.md
```

File: tools/packages/taipy-rest/MANIFEST.in [hunk 4]
```diff
@@ -1,3 +1,4 @@
 include taipy/rest/*.json
 include *.json
 include taipy/rest/setup.requirements.txt
+include package_desc.md
```
Comment (jrobinAV) [inline tools/packages/taipy-rest/MANIFEST.in:4]:
```suggestion
include taipy/rest/package_desc.md
```

File: tools/packages/taipy-templates/MANIFEST.in [hunk 5]
```diff
@@ -1,3 +1,4 @@
 recursive-include taipy/templates *
 include *.json
 include taipy/templates/setup.requirements.txt
+include package_desc.md
```
Comment (jrobinAV) [inline tools/packages/taipy-templates/MANIFEST.in:4]:
```suggestion
include taipy/templates/package_desc.md
```

**Review by jrobinAV — APPROVED** [review]

**Review by FabienLelaquais — APPROVED** [review]
Well done!

**Pull request merged**